.github/workflows/workflow.yml

```diff
@@ -0,0 +1,153 @@
+name: Country Template
+
+on: [ push ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7.12
+      - name: Cache build
+        id: restore-build
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ github.ref }}${{ env.pythonLocation }}-build-${{ hashFiles('setup.py') }} # Cache the entire build Python environment
+      - name: Build package
+        if: steps.restore-build.outputs.cache-hit != 'true'
+        run: make build
```

Comment:
Won't that leave the built package in its very first state, never updating the cache? 🤔

Reply:
The cache is updated if any element of the cache key is changed, in this case the `setup`, the `branch` or the `python env`. Do you think something else should be added?

Reply:
Yes, you're right. We did encounter the problem in `Openfisca-France`. I opened a [PR](https://github.com/openfisca/openfisca-france/pull/1685) to solve it.

Reply:
I will integrate the changes here as well.